ddlambda_test.go

```diff
@@ -11,12 +11,16 @@ import (
 	"context"
 	"net/http"
 	"net/http/httptest"
+	"os"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 )
 
 func TestInvokeDryRun(t *testing.T) {
+	os.Setenv(UniversalInstrumentation, "false")
+	os.Setenv(DatadogTraceEnabledEnvVar, "false")
```

Comment:
Use `t.Setenv` instead. It will automatically cleanup after itself.

Reply:
Thanks!